### ISSUE TITLE
List Nav Bar is not being Refreshed

### ISSUE THREAD

**Issue by LordLambda** [rawcodeio/Bug-Report#17]
The Right-Side Nav bar for lists doesn't update right away when you add a new list.
Example Screencast [here][screencastlink] (sorry for flash. I need a better screen-recorder.) (also I think the menu icon becoming desynced 0:36 was a local browser issue, but might be something to look into).

Needless to say if you look regardless of Private/Public New lists that are created are being cached/not updated (I think it's a two-fur). One part it doesn't refresh the list view on the right to show the new lists.

Second off it keeps the text cached for when creating a new list again, which could lead to some confusion.

Tested on my normal suite of browsers:
Mac Chrome (Latest)
Mac Firefox (Latest)
Mac Safari (9.0.1)
Windows 7 IE 10
Windows 7 IE 11
Windows 10 Microsoft Edge
Arch Linux: Firefox (Latest)

Keep being awesome though. :+1: You're pretty fast at fixing them.

[screencastlink]: http://screencast.com/t/hZFkd5f7

**Comment by BenMann** [comment]
True! And strange... this should be ajax, so I should be able to refresh the UI easily or at least refresh on finish — looking into it. Thanks for the report and the kind words. You're also awesome at finding them :smile:
